game.py

```diff
@@ -1,52 +1,23 @@
+import random
+from  wonderwords import RandomWord
+#https://pypi.org/project/wonderwords/
```

Comment:
All of the random operations are performed out in the `main.py`. We shoudn't need either of these imports here.